sql/moz-fx-data-shared-prod/firefox_ios/ad_activation_performance/view.sql

```diff
@@ -0,0 +1,46 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_ios.ad_activation_performance`
+AS
+WITH client_activation_per_campaign AS (
+  SELECT
+    DATE(submission_timestamp) AS `date`,
```

Comment:
Is this right? We have a `submission_timestamp` in `firefox_ios_clients`? What does it represent?